tests/test_store.py

```diff
@@ -214,6 +220,34 @@ def test_view_query(
         # tests that the query was valid and successful, that a view subset has actually been retrived
         assert synapse_store_special_scope.storageFileviewTable.empty is False
 
+    @pytest.mark.parametrize(
+        "asset_view,columns,expectation",
+        [
+            ("syn62339865", ["path"], pytest.raises(ValueError)),
+            ("syn62340177", ["id"], pytest.raises(ValueError)),
+            ("syn23643253", ["id", "path"], does_not_raise()),
+        ],
+    )
+    def test_view_query_exception(
```

Comment:
when I read `test_view_query`, I think there are certain cases that are not covered: 
1) when project scope is None but columns, where_clauses are provided
2) when columns is None but where_clauses and project scope are provided. 

Should they also be covered?

Reply:
@linglp yes thats a good point thanks!